test/db/esil/x86_64

```diff
@@ -8,7 +8,7 @@ e asm.bits=64
 EOF
 EXPECT=<<EOF
 imul r9
-0x00000000 64,rdx,~,64,rax,~,*,DUP,r9,=,r9,-,?{,1,1,}{,0,0,},cf,:=,of,:=
+0x00000000 64,64,r9,~,64,rax,~,*,>>,rdx,=,r9,rax,*=,64,64,rax,~,>>,rdx,-,?{,1,1,}{,0,0,},cf,:=,of,:=
```

Comment:
Ehhm, this looks funny. But I am not sure what kind of funny :D
Have you checked it?